vnext/Microsoft.ReactNative/ReactApplicationDelegate.cpp

```diff
@@ -20,10 +20,28 @@ using namespace Windows::ApplicationModel::Activation;
 
 namespace winrt::Microsoft::ReactNative::implementation {
 
-static void ApplyArguments(ReactNative::ReactNativeHost const & /*host*/, std::wstring const &arguments) noexcept {
-  // Microsoft::ReactNative::implementation::ReactNativeHost* hostImpl {
-  // get_self<Microsoft::ReactNative::implementation::ReactNativeHost>(host)};
+static void ApplyArguments(ReactNative::ReactNativeHost const &host, std::wstring const &arguments) noexcept {
+  Microsoft::ReactNative::implementation::ReactNativeHost *hostImpl{
+      get_self<Microsoft::ReactNative::implementation::ReactNativeHost>(host)};
   if (!arguments.empty() /*&& host.HasInstance()*/) {
+    constexpr wchar_t delimiter = L',';
+    std::wistringstream argumentStream(arguments);
+    std::wstring token;
+    while (std::getline(argumentStream, token, delimiter)) {
+      if (token == L"-?") {
+        std::cout << "Options:\n"
+                     "  directDebugging,<port>    Enable direct debugging on specified port.\n";
+        std::cout.flush();
+      } else if (token == L"directDebugging") {
```

Comment:
let's keep the format of options uniform, if we're using -? then this should be --direct-debugging?

Reply:
oh I see we have the -- options and then these that don't have the --. I'm not sure when to use one versus the other, can you clarify?

Reply:
My bad.
The options starting with -- are options for the react native windows CLI, while this option (`directDebugging`) is for any app built with react native for windows (using Microsoft.ReactNative). I saw another option `remoteDebugging` passed to the app in the deploy/start powershell script so I went with camel case for the new option. It seems that that `remoteDebugging` is not being used anymore, so it might be safe to remove?
Either way, I'm not sure what's the best way to proceed. Perhaps make it consistent across the CLI and the apps? What do you suggest?